### ISSUE TITLE
Popover Typescript definition missing

### ISSUE THREAD

**Issue by V-ed** [bestguy/sveltestrap#227]
While Popover is available through javascript using definitions from the package :

https://github.com/bestguy/sveltestrap/blob/934dca68441899eaafd378f5d7186e74b1a03267/src/index.js#L64

The definition is not present in `index.d.ts` : https://github.com/bestguy/sveltestrap/blob/934dca68441899eaafd378f5d7186e74b1a03267/src/index.d.ts

---

A quick fix until then is to add a file under `./@types/sveltestrap/index.d.ts` (or any folder*) that contains this :

```typescript
import Popover from 'sveltestrap/src/Popover';

declare module 'sveltestrap' {
	export { Popover };
}
```

and to include types under tsconfig : `"include": [..., "./@types/**/*.ts"],` (*switch the folder here if you went with something else than `@types`)

---

Thanks for this useful library, using bootstrap in svelte using this is much easier!

**Comment by bestguy** [comment]
Ah, oops!  Sorry we missed.  There is a PR open to fix (#228) but pending a question

**Comment by V-ed** [comment]
No worries, thanks for the quick response, will be updating once it's released with that PR! :)

**Comment by bestguy** [comment]
Hi @V-ed this is released in [sveltestrap@3.12.2](https://github.com/bestguy/sveltestrap/releases/tag/v3.12.2)

Please confirm this corrects your issue and feel free to close issue if so.

**Comment by V-ed** [comment]
It does indeed, thanks for the quick fix!
